examples_tests/02.ComputeShader/main.cpp

```diff
@@ -1,152 +1,225 @@
 #define _NBL_STATIC_LIB_
 #include <nabla.h>
 
-#include "../../source/Nabla/COpenGLDriver.h"
-
+#include "../common/CommonAPI.h"
 
 using namespace nbl;
 using namespace core;
 
 
+const char* src = R"(#version 450
+
+layout (local_size_x = 16, local_size_y = 16) in;
+
+layout(push_constant) uniform pushConstants {
+    layout (offset = 0) uvec2 imgSize;
+} u_pushConstants;
+
+layout (set = 0, binding = 0, rgba8) uniform readonly image2D inImage;
+layout (set = 0, binding = 1, rgba8) uniform image2D outImage;
+
+void main()
+{
+	if (all(lessThan(gl_GlobalInvocationID.xy, u_pushConstants.imgSize)))
+	{
+		vec3 rgb = imageLoad(inImage, ivec2(gl_GlobalInvocationID.xy)).rgb;
+		
+		imageStore(outImage, ivec2(gl_GlobalInvocationID.xy), vec4(1, 1, 0, 1));
+	}
+})";
+
+
+int getQueueFamilyIndex(const core::smart_refctd_ptr<nbl::video::IPhysicalDevice>& gpu, uint32_t requiredQueueFlags)
+{
+	auto props = gpu->getQueueFamilyProperties();
+	int currentIndex = 0;
+	for (const auto& property : props)
+	{
+		if (property.queueFlags & requiredQueueFlags)
```

Comment:
tjhis should be `(property.queueFlags & requiredQueueFlags) == requiredQueueFlags`

Reply:
Yeah, missed that